tests/test_interface.py

```diff
@@ -6,7 +6,7 @@ def test_init():
     """
     Smoke test just to initialize empty interface state.
     """
-    interface_state = InterfaceState()
+    interface_state = InterfaceState(print)
```

Comment:
To by chtelo dopsat nejaky testik, ze takto prirazena funkce dela opravdu to, co ma.

Reply:
pokusím se :)